pkg/lib/timeformat.js

```diff
@@ -16,9 +23,24 @@ export const time = t => formatter({ timeStyle: "short" }).format(t);
 export const timeSeconds = t => formatter({ timeStyle: "medium" }).format(t);
 // June 30, 2021
 export const date = t => formatter({ dateStyle: "long" }).format(t);
+// 2021-06-30
+export const dateShort = t => formatter().format(t);
 // Jun 30, 2021, 7:41 AM
 export const dateTime = t => formatter({ dateStyle: "medium", timeStyle: "short" }).format(t);
 // Jun 30, 2021, 7:41:23 AM
 export const dateTimeSeconds = t => formatter({ dateStyle: "medium", timeStyle: "medium" }).format(t);
 // Wednesday, June 30, 2021
 export const weekdayDate = t => formatter({ dateStyle: "full" }).format(t);
+
+// The following options are helpful for placeholders
+// yyyy/mm/dd
+export const dateShortFormat = () => locales[dateFormatLangDateFns()].formatLong.date({ width: 'short' });
+
+// Parse a string localized date like 30.06.21 to a Date Object
+export function parseShortDate(dateStr) {
+    const parsed = parse(dateStr, dateShortFormat(), new Date());
```

Comment:
I'm a bit surprised that this does not need a locale specification? This tries to parse the date for any existing locale?